protocol-specs/generator_outputs/lisk_codec/validBytesEncodings.json

```diff
@@ -0,0 +1,78 @@
+{
+	"title": "Valid bytes encodings",
+	"summary": "Examples of encoding bytes as required by lisk-codec",
+	"config": {
+		"network": "devnet"
+	},
+	"runner": "lisk_codec",
+	"handler": "validBytesEncodings",
+	"testCases": [
+		{
+			"description": "Encoding of bytes types",
+			"config": {
+				"network": "devnet"
+			},
+			"input": {
+				"bytes": {
+					"object": {
+						"address": {
+							"type": "Buffer",
+							"data": [
+								225,
+								26,
+								17,
+								54,
+								71,
+								56,
+								34,
+								88,
+								19,
+								248,
+								110,
+								168,
+								82,
+								20,
+								64,
+								14,
+								93,
+								176,
+								141,
+								110
+							]
+						}
+					},
+					"schema": {
+						"type": "object",
+						"properties": {
+							"address": {
+								"dataType": "bytes",
+								"fieldNumber": 1
+							}
+						}
+					}
+				},
+				"emptyBytes": {
+					"object": {
+						"address": {
+							"type": "Buffer",
```

Comment:
there is no Buffer type, maybe you meant `"dataType": "bytes"?`

Reply:
"Buffer" is how JSON.stringify serialises the buffer I think there's nothing we can do about it.

Reply:
maybe encode it as `buffer.toString('binary')`?

Reply:
I misunderstood what the Buffer was, clarified from Pablo, Buffer is fine